packages/client/src/helpers/PeerConnectionHandler.ts

```diff
@@ -0,0 +1,86 @@
+import { detectAudioLevels } from './detect-audio-levels';
+import { isReactNative } from './platforms';
+import { SoundStateChangeHandler } from './sound-detector';
+
+export class PeerConnectionHandler {
+  private pc1: RTCPeerConnection | undefined;
+  private pc2: RTCPeerConnection | undefined;
+
+  constructor() {
+    this.initializePeerConnection();
+  }
+
+  /**
+   * Internal method to initialize the peer connections.
+   */
+  private async initializePeerConnection() {
+    if (!isReactNative()) return;
```

Comment:
I think these `isReactNative()` checks don't belong to this class. If we name it right, it will be obvious this class is meant to be run in a RN context.

On a higher level, we anyway create this instance only if running in RN.